src/artemis/parameters/external_parameters.py

```diff
@@ -169,8 +171,12 @@ def from_dict(cls, dict_params: dict[str, dict]):
             base_uri=f"{path.as_uri()}/",
             referrer=True,
         )
-        # TODO improve failed validation error messages
-        jsonschema.validate(dict_params, full_schema, resolver=resolver)
+        try:
+            jsonschema.validate(dict_params, full_schema, resolver=resolver)
+        except ValidationError:
+            artemis.log.LOGGER.error("Invalid json parameters")
+            raise ValidationError("Invalid Json parameters")
```

Comment:
should: It's nicer to just use `raise` here, which will reraise the original exception including the stack trace and the original place the validation failed.

Reply:
Actually it looks like that validate function does the same error handling anyway, so I've removed this part